cmd/apiserver_receive_adapter/main.go

```diff
@@ -18,24 +18,32 @@ package main
 
 import (
 	"flag"
-	"log"
+	"fmt"
 
-	// Uncomment the following line to load the gcp plugin (only required to authenticate against GKE clusters).
+	// Uncomment the following line to load the gcp plugin
+	// (only required to authenticate against GKE clusters).
 	// _ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 
+	"knative.dev/eventing/pkg/reconciler/apiserversource/resources"
+
```

Comment:
remove whitespace and run go importer formatting

Reply:
done